.github/workflows/assign_bugs.yml

```
@@ -0,0 +1,14 @@
+name: "Set Bug Issue Label and Assignee"
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  autoassign:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Naturalclar/issue-action@v2.0.2
+        with:
+          title-or-body: "both"
+          parameters: '[ {"keywords": ["[BUG]", "bug"], "labels": ["Bug", "BUG"], "assignees": ["antalszava"]}]'
```

Should this be instead
```suggestion
          parameters: '[ {"keywords": ["[BUG]", "bug"], "labels": ["bug :bug:"], "assignees": ["antalszava"]}]'
```
?

Since the name of our label is `bug :bug:` (where `:bug:` is 🐛 )

---

Great catch, thank you!

1. Modified the label in the trial repo and tried auto label assignment with a new issue (https://github.com/antalszava/bugs-scan/issues/9). The label was indeed not applied.
2. https://github.com/antalszava/bugs-scan/commit/c5593e9b3b8a511435eff7b4732085f215f3a313 updated the `"labels"` to use the custom bug label `"bug :bug:"`.
3. Labelling a new bug issue https://github.com/antalszava/bugs-scan/issues/10 worked as expected. :100: